src/Evolution/Systems/Cce/WorldtubeBufferUpdater.cpp

```diff
@@ -39,9 +39,11 @@ std::pair<size_t, size_t> create_span_for_time_value(
       lower_bound < upper_bound,
       "The supplied `lower_bound` is greater than `upper_bound`, which is not "
       "permitted");
-  ASSERT(2 * interpolator_length + pad < upper_bound,
+  ASSERT(2 * interpolator_length + pad <= upper_bound,
```

Comment:
I can't figure out what this function does, but this new check is what is required to avoid underflow in the math, so it's probably correct.

Reply:
Yea, I traced this back a decent bit through the code, never fully understood it but understood it well enough to expect several other ASSERTs to trigger if it was wrong :)